android/src/app/organicmaps/widget/menu/NavMenu.java

```diff
@@ -9,22 +10,24 @@
 
 import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+
 import com.google.android.material.bottomsheet.BottomSheetBehavior;
+
+import java.text.DateFormat;
```

Comment:
Why imports are unsorted now?

Reply:
Sorry there were warning and I used the shortcut to optimise imports which automatically rearranged them. I won't do it again in this project